_includes/why_d4a.md

```diff
@@ -0,0 +1,113 @@
+The [Docker for
+{{cloudprovider}}](https://docs.docker.com/docker-for-{{cloudprovider |
+downcase}}/) project was created and is being actively developed to ensure that
+Docker users can enjoy a fantastic out-of-the-box experience on
+{{cloudprovider}}.  It is now generally available and can now be used by
+everyone.
+
+As an informed user, you might be curious to know what this project has to offer
+you for running your development, staging, or production workloads.
+
+## Native to Docker
+
+Docker for {{cloudprovider}} was built with a passion for simplicity and great
+user experience. We believe strongly that additional complexity must be
+justified by substantial additional benefit.  Some solutions are very complex to
+operate or layer unneeded additional APIs on top of Docker, but Docker for
+{{cloudprovider}} provides a Docker-native solution.
+
+Instead of having to learn one or more layers on top of Docker, with Docker for
```

Comment:
Docker for ____ allows you to interact with Docker directly (including native Docker orchestration), instead of distracting you with the need to navigate extra layers on top of Docker.